tests/functional/pv/pv_services/test_cephfs_rwx_pvc_ec_pool_lifecycle.py

```
@@ -0,0 +1,137 @@
+        )
+        log.info(f"PVC '{pvc_obj.name}' created and bound")
+
+        # Get subvolume name for later verification
+        subvol_name = pvc_obj.get_cephfs_subvolume_name
+        log.info(f"CephFS subvolume: {subvol_name}")
+
+        # Get the image UUID from the PV volume handle
+        pv_obj = pvc_obj.backed_pv_obj
+        volume_handle = pv_obj.get()["spec"]["csi"]["volumeHandle"]
+        image_uuid = volume_handle.split("-", 4)[-1]
+        log.info(f"Volume handle: {volume_handle}, image UUID: {image_uuid}")
```

Ruff (0.15.20)

[warning] Logging statement uses f-string

(G004)